scripts/seed-prestate.sh

```
@@ -261,4 +261,34 @@ for c in "${CASES[@]}"; do
 
+# ---------- propagation wait ----------
+#
+# gh issue list --label X queries GitHub's search index which lags issue
+# creation by 2-15s. vr_dedup_init reads from this index, so an audit that
+# fires immediately after this script may see an empty cache and skip
+# Phase 2 entirely. Poll until the index reflects the issues we just created.
+
+if [ "$DRY_RUN" = "false" ]; then
+    expected="${#CASES[@]}"
+    actual=""
+    for attempt in 1 2 3 4 5 6 7 8 9 10; do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)